backend/src/API/devPortfolioAPI.ts

```diff
@@ -40,8 +40,8 @@ export const createNewDevPortfolio = async (
   instance: DevPortfolio,
   user: IdolMember
 ): Promise<DevPortfolio> => {
-  const canCreateDevPortfolio = await PermissionsManager.isLeadOrAdmin(user);
```

Comment:
Personally I don't see the need to change all these to `isLeadOrAdmin`. `canCreateDevPortfolio` is fine and is more descriptive. But I'll leave it up to you.

Reply:
I think the method names are descriptive enough, and changing it to `isLeadOrAdmin` keeps it consistent with the rest of the original file.